src/sass/storage/svg/test.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="512" height="512" viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg"
```

Comment:
maybe add second sprite. to test spriteabylity?

Reply:
added 2 test svg, works good